src/Core/src/Platform/iOS/MauiUIApplicationDelegate.cs

```diff
@@ -166,5 +170,18 @@ public virtual void PerformFetch(UIApplication application, Action<UIBackgroundF
 		public IServiceProvider Services { get; protected set; } = null!;
 
 		public IApplication Application { get; protected set; } = null!;
+
+		void ViewDidMoveToWindow(IntPtr @this)
+		{
+			UIView nsObject = (UIView)ObjCRuntime.Runtime.GetINativeObject(@this, true, typeof(UIView))!;
+
+			using (var orig = didMoveToWindow?.Restore())
```

Comment:
It seems wasteful to create  new object (+delegate) for every call, maybe the delegate to call the original method should be on the Swizzle instance instead (potentially only initialized on first use)?